Makefile

```
@@ -151,6 +151,8 @@ core: $(CACHE)/.setup
 
 apps: $(CACHE)/.setup
 	@$(ECHO) $(PRINT_RECIPE)
+	@cd $(SRC_DIR)/batteryMonitorUI && BUILD_DIR="$(PACKAGES_APP_DEST)/Battery monitor tool/App/BatteryMonitorUI" make
+    @find $(SRC_DIR)/batteryMonitorUI -depth -type d -name res -exec cp -r {}/. "$(PACKAGES_APP_DEST)/Battery monitor tool/App/BatteryMonitorUI/res/" \;
```

```suggestion
	@find $(SRC_DIR)/batteryMonitorUI -depth -type d -name res -exec cp -r {}/. "$(PACKAGES_APP_DEST)/Battery monitor tool/App/BatteryMonitorUI/res/" \;
```

the spaces here break the build